pkg/jwt/configuration.go

```diff
@@ -102,3 +112,39 @@ func NewSignatureValidatorFromJSONWebKeySet(jwks *jose.JSONWebKeySet) (Signature
 
 	return NewDemultiplexingSignatureValidator(namedSignatureValidators, allSignatureValidators), nil
 }
+
+func NewSignatureValidatorFromJSONWebKeySetFile(path string, refreshInterval time.Duration) SignatureValidator {
+	// Hmm. I don't want us to first read the file once, to initialize this, and then read the file periodically to update it.
+	// However, I also don't want us to initialize this in a state that isn't ready for use.
+	validator := NewForwardingSignatureValidator(nil)
+
+	// TODO: Run this as part of the program.Group, so that it gets
+	// cleaned up upon shutdown.
+	go func() {
+		t := time.NewTicker(refreshInterval)
+		for {
```

Comment:
nit: `for _ = range t.C` might be a bit more idiomatic

Reply:
Done